src/share/mkjail/create.sh

```diff
@@ -80,10 +80,9 @@ if [ x"${fflag}" = x1 ] && [ ! -d /var/db/mkjail/flavours/${FLAVOUR} ]; then
 fi
 
 # Create the ZFS filesystem
-echo "Creating ${ZPOOL}/jails/${JAILNAME}..."
-zfs create -p -o mountpoint=/jails ${ZPOOL}/jails
-zfs create -p ${ZPOOL}/jails/${JAILNAME}
-zfs set mkjail:version=${VERSION} ${ZPOOL}/jails/${JAILNAME}
+echo "Creating ${ZPOOL}${JAILDATASET}/${JAILNAME}..."
+zfs create "${ZPOOL}${JAILDATASET}/${JAILNAME}"
+zfs set mkjail:version="${VERSION}" "${ZPOOL}${JAILDATASET}/${JAILNAME}"
```

Comment:
I think `VERSION` is in quotes to allow greater flexibility in version names.

Reply:
I added the quotes 😀